iac-extensions/kubernetes/src/test/java/org/sonar/iac/kubernetes/checks/KubernetesVerifier.java

```diff
@@ -72,38 +75,53 @@ public class KubernetesVerifier {
 
   private static final Logger LOG = LoggerFactory.getLogger(KubernetesVerifier.class);
   public static final Path BASE_DIR = Paths.get("src", "test", "resources", "checks");
-  private static final YamlParser YAML_PARSER = new YamlParser();
+  private static final SensorContextTester SENSOR_CONTEXT = SensorContextTester.create(BASE_DIR.toAbsolutePath());
+  private static final KubernetesAnalyzer KUBERNETES_ANALYZER = initializeKubernetesAnalyzer();
+  private static final TreeParser<Tree> PARSER = KUBERNETES_ANALYZER::parse;
 
-  public static void verify(String templateFileName, IacCheck check) {
+  public static void verify(String templateFileName, IacCheck check, String... fileNames) {
+    HelmInputFileContext inputFileContext;
+    BiConsumer<Tree, Map<Integer, Set<Comment>>> commentsVisitor;
     if (containsHelmContent(templateFileName)) {
```

Comment:
I tried to utilize the check k8s/helm that we have inside the KubernetesAnalyzer, but the logic for comments visitors is different in these cases